Assets/NuGet/Editor/NugetPackageSource.cs

```diff
@@ -77,6 +77,11 @@ public bool HasPassword
         /// </summary>
         public bool IsLocalPath { get; private set; }
 
+        /// <summary>
+        /// Get or sets a value indicating if this is a local path that is 3.3+;
+        /// </summary>
+        public bool IsLocalPathAndVersion33 { get; set; }
```

Comment:
Should be a `private set`, and I'd recommend making this just track the IsVersion33 test. As implemented this can diverge from 'IsLocalPath'. You could write a property to return the result of these two state properties.